internal/cmd/login.go

```diff
@@ -143,6 +145,8 @@ func login(cli *CLI, options loginCmdOptions) error {
 	case options.AccessKey != "":
 		loginReq.AccessKey = options.AccessKey
```

Comment:
Should we output something in the case of an access key environment variable too? Might make sense where it takes precedence

Reply:
I can't really come up with anything great either, maybe just `Logging in with access key` would suffice.